cmd/install/install_helm.go

```diff
@@ -0,0 +1,153 @@
+package install
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift/hypershift/pkg/version"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var helmTemplateParams = TemplateParams{
+	Namespace:                   ".Release.Namespace",
+	HyperShiftImage:             ".Values.image",
+	OIDCS3Name:                  ".Values.oidc.s3.name",
+	OIDCS3Region:                ".Values.oidc.s3.region",
+	OIDCS3CredsSecret:           ".Values.oidc.s3.credsSecret",
+	OIDCS3CredsSecretKey:        ".Values.oidc.s3.credsSecretKey",
+	AWSPrivateRegion:            ".Values.aws.private.region",
+	AWSPrivateCredsSecret:       ".Values.aws.private.credsSecret",
+	AWSPrivateCredsSecretKey:    ".Values.aws.private.credsSecretKey",
+	ExternalDNSCredsSecret:      ".Values.externaldns.credsSecret",
+	ExternalDNSDomainFilter:     ".Values.externaldns.domainFilter",
+	ExternalDNSTxtOwnerID:       ".Values.externaldns.txtOwnerId",
+	ExternalDNSImage:            ".Values.externaldns.image",
+	RegistryOverrides:           ".Values.registryOverrides",
+	AROHCPKeyVaultUsersClientID: ".Values.azure.keyVault.clientId",
+	TemplateNamespace:           false,
+	TemplateParamWrapper: func(name string) string {
+		return fmt.Sprintf("{{ %s }}", name)
+	},
+}
+
+func NewHelmRenderCommand(opts *Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "helm",
+		Short:        "Generate a Helm chart for the HyperShift Operator",
+		SilenceUsage: true,
+	}
+
+	cmd.Flags().StringVar(&opts.OutputFile, "output-dir", "", "File to write the rendered manifests to. Writes to STDOUT if not specified.")
```

Comment:
nit: The description is not accurate

Reply:
also, since you are defaulting to `./chart`, it would make sense to set that as the default value of opts.OutputFile and pass it to StringVar

Reply:
both adressed. ty